src/Cms/Language.php

```diff
@@ -461,6 +462,7 @@ public function save(): static
 	{
 		try {
 			$existingData = Data::read($this->root());
+			$data['translations'] = $this->translations()->load($existingData['translations'] ?? []);
```

Comment:
Where is `$data` coming from? Does this need to be `$existingData['translations']`?